RecoTracker/LSTCore/src/alpaka/Event.dev.cc

```diff
@@ -0,0 +1,1668 @@
+#include "HeterogeneousCore/AlpakaInterface/interface/memory.h"
+
+#include "Event.h"
+
+using Device = ALPAKA_ACCELERATOR_NAMESPACE::Device;
+using Queue = ALPAKA_ACCELERATOR_NAMESPACE::Queue;
+using Acc1D = ALPAKA_ACCELERATOR_NAMESPACE::Acc1D;
+using Acc3D = ALPAKA_ACCELERATOR_NAMESPACE::Acc3D;
+
+using namespace ALPAKA_ACCELERATOR_NAMESPACE::lst;
+
+void Event::initSync(bool verbose) {
+  alpaka::wait(queue);  // other calls can be asynchronous
```

Comment:
This function (and the sync) is needed in the standalone configuration, where the same `LSTEvent` object is reused for different events. It can indeed be avoided in the CMSSW configuration, and we have updated the code to not use this, in accordance with your suggestions in this https://github.com/cms-sw/cmssw/pull/45117#discussion_r1749987469.